lib/algo/memoization/test.js

```diff
@@ -0,0 +1,31 @@
+const chai = require('chai');
+const knapsack = require('./0-1 Knapsack');
+
+class Item {
+  constructor(profit, volume) {
+    this.profit = profit;
+    this.volume = volume;
+  }
+}
+
+const expect = chai.expect;
+
+// var items = [...new Array(9)].map(() => new Item(Math.round(Math.random() * 10) + 1, Math.round(Math.random() * 8) + 1))
+describe('knapsack', () => {
+  const items = [
+    new Item(12, 2),
+    new Item(5, 4),
+    new Item(1, 5),
+    new Item(56, 23),
+    new Item(2, 6),
+    new Item(1, 5),
+    new Item(12, 8),
+    new Item(2, 3)
+  ];
+  const capacity = 7;
```

Comment:
Please move `const items` and `const capacity` to `it` block, as they should vary for each test case.

If you consider item to be constant for all the test cases, please take it out of `describe`.